geth/api/backend.go

```diff
@@ -251,16 +253,32 @@ func (b *StatusBackend) getVerifiedAccount(password string) (*account.SelectedEx
 	return selectedAccount, nil
 }
 
-// ApproveSignRequest instructs backend to complete sending of a given transaction
-func (b *StatusBackend) ApproveSignRequest(id string, password string) sign.Result {
-	return b.pendingSignRequests.Approve(id, password, b.getVerifiedAccount)
+// prepareTxArgs given gas and gasPrice will prepare a valid sign.TxArgs.
+func (b *StatusBackend) prepareTxArgs(gas, gasPrice int64) (args sign.TxArgs) {
```

Comment:
It does not use `StatusBackend` so it can be actually a function.

Reply:
Also, if that can be a function, maybe we can put in into a separate file `txargs.go`?